src/app/static/src/js/utils/types.ts

```diff
@@ -128,4 +128,17 @@ export interface ChildCustomStyle {
 export interface Error {
     code: string;
     message: string | null;
+}
+
+export interface WorkflowReportWithDependency {
+    name: string,
+    instance?: string
+    params?: Record<string, string>,
+    depends_on?: string[]
+}
+
+export interface Dependency {
```

Comment:
As suggested above, this seems a confusing name too, as it's much more than just dependencies. `WorkflowSummaryDetails`?

Reply:
@Nicolas-Dolan  actually changed the name to `workflowSummary`, I will use that instead.